packager/live_packager_export.cc

```diff
@@ -19,6 +19,10 @@ LivePackager_t livepackager_new(LivePackagerConfig_t cfg) {
       .segment_number = cfg.segment_number,
       .m2ts_offset_ms = cfg.m2ts_offset_ms,
       .timed_text_decode_time = cfg.timed_text_decode_time,
+      .decryption_key =
```

Comment:
If the decryption key and key id are unconditionally set, looks like decryption will always occur, even if the source is unencrypted.

Re: 
```
if (!config_.decryption_key.empty() && !config_.decryption_key_id.empty())
```